### ISSUE TITLE
[Adaptive Cards - Main Body Content of Adaptive Card]: There is no underline or chevron is present for the 'Create reusable card templates' link.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8674]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 23H2 (OS Build 25931.1000)
Browser Version: Edge Dev (118.0.2060.1)

**Repro Steps:**
1. Open the above URL on the edge dev browser.
2. Press tab key and navigate to links present on the screen and observe the issue.

**Actual Result:**
While navigating to the 'Links' present on the screen, there is no underline or chevron is present for the 'Create reusable card templates' link.

**Expected Result:**
There should be underline or chevron should be present for the 'Create reusable card templates' link.

**Note:**
Same issue is repro throughout the screen for the links.

**User Impact:**
It will impact the low vision user if there is no underline present for the links due to this they will not able to differentiate whether it is a link or plain text.

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

![Usable - There is no underline is present for the create reusale card templates](https://github.com/microsoft/AdaptiveCards/assets/95913748/f68b8c47-d46e-4cf4-8cc8-a1259329cbf9)

https://github.com/microsoft/AdaptiveCards/assets/95913748/11474acf-646a-47b0-97d8-8f0058e60e06

### Card JSON

```JSON
NA
```
